variants/tests/util.hpp

```diff
@@ -8,8 +8,8 @@
 // SPDX-FileCopyrightText: Michael Park
 // SPDX-License-Identifier: BSL-1.0
 
-#ifndef _UTIL_HPP
-#define _UTIL_HPP
+#ifndef KOKKOS_UTIL_HPP
+#define KOKKOS_UTIL_HPP
```

Comment:
I would rename this include guard to make it more specific
```suggestion
#ifndef KOKKOS_VARIANT_TEST_UTIL_HPP
#define KOKKOS_VARIANT_TEST_UTIL_HPP
```